src/mono/mono/metadata/appdomain.h

```diff
@@ -106,6 +106,9 @@ mono_domain_foreach        (MonoDomainFunc func, void* user_data);
 MONO_API MONO_RT_EXTERNAL_ONLY MonoAssembly *
 mono_domain_assembly_open  (MonoDomain *domain, const char *name);
 
+MONO_API void
+mono_domain_ensure_entry_assembly (MonoDomain *domain, MonoAssembly *assembly);
```

Comment:
I guess the wasm runtime can't use private headers? If so, this is fine.

Reply:
Yeah, I don't believe you can.  I get a compiler error.